src/components/Slider/Slider-test.js

```diff
@@ -15,13 +14,8 @@ describe('Slider', () => {
         min={0}
         max={100}
         step={1}
-        onChange={mockFn}>
-        <TextInput
```

Comment:
Any reason removing `<TextInput>` here...?

Reply:
Maybe I'm wrong but I didn't see any code or documentation where that is the proper usage of `<Slider />`. It looks like the `TextInput` gets rendered [as part of the component](https://github.com/carbon-design-system/carbon-components-react/blob/master/src/components/Slider/Slider.js#L382) and is not mentioned anywhere [in the Story](https://github.com/carbon-design-system/carbon-components-react/blob/master/src/components/Slider/Slider-story.js#L16). I thought it was a mistake, but since I'm new to this project perhaps I misunderstood.

Reply:
@tw15egan Wondering if this is something you can chime in...? Thanks!

Reply:
Yeah, probably just an oversight in the original creation of the component. The `TextInput` is rendered by the component itself, so not needed in the tests